node/src/accountant/db_big_integer/big_int_db_processor.rs

```diff
@@ -146,14 +166,17 @@ impl<T: TableNameDAO> UpdateOverflowHandler<T> for UpdateOverflowHandlerReal<T>
 }
 
 impl<T: TableNameDAO + Debug> UpdateOverflowHandlerReal<T> {
-    fn execute_update<'a>(
-        conn: Either<&dyn ConnectionWrapper, &Transaction>,
-        config: &BigIntSqlConfig<'a, T>,
+    fn execute_update<'params>(
+        conn: Either<&dyn ConnectionWrapper, &dyn TransactionWrapper>,
+        config: &BigIntSqlConfig<'params, T>,
         execute_params: &[(&str, &dyn ToSql)],
     ) {
-        match BigIntDbProcessor::<T>::prepare_statement(conn, config.overflow_update_clause)
-            .execute(execute_params)
-            .expect("logic broken given the previous non-overflow call accepted right")
+        match BigIntDatabaseProcessorReal::<T>::prepare_statement(
+            conn,
+            config.overflow_update_clause,
+        )
+        .execute(execute_params)
+        .expect("logic broken given the previous non-overflow call accepted right")
```

Comment:
Maybe you can use this statement, in case you like it...

```
If the previous non-overflow call was accepted, this should not happen. There is some flaw in logic.
```